assets/Meow/DynamicAudio.gd

```diff
@@ -5,17 +5,30 @@ var musicOn = true
 var masterVol = -10
 # Called when the node enters the scene tree for the first time.
 func _ready():
-	var dirMusic = "res://Meow/Music/Reg/"
-	#dir_contents(dirMusic)
+	var dirMusic = "res://Meow/Music/Reg"
 	var player = AudioStreamPlayer.new()
 	self.add_child(player)
 	player.volume_db = masterVol
-	var playlist = dir_counter2(dirMusic)
+	var playlistFiles = dir_counter(dirMusic,".import")
+	var playlistPaths = assetsPaths(playlistFiles,"path=\"")
```

Comment:
I think this code is fine to merge as-is, but I think it would be a little cleaner if we didn't have to step through the files twice; is there any way we can get the asset paths the first time we process each of the files? 

It would be nice to have a function where we could pass one file path that we care about and get the correct "asset path" back.

Reply:
i pass all the files in a folder which gets the imports that are there. 
Second function gets the paths of the assets of the imports by reading the import files

reason they are separated is due to the fact someone else might need to use these functions possibly. i could combine them into one but thought this might be useful.